### ISSUE TITLE
Add to readme how to setup CI=true flag with Ember.js >= v3.0

### ISSUE THREAD

**Issue by raido** [danlynn/ember-cli#17]
As of Ember-CLI v3.0, default generated `config/target.js` has `process.env.CI` check to determine if IE11 should be considered as build target.

http://www.ember-cli-diff.org/?#/v2.18.2/v3.0.0

Since it would not make sense to include `export CI=true `in `.bashrc` as default because it would affect also development builds, then best advice probably is to add CI flag just as `docker run --env CI=true...`

So basically README.md should be updated to reflect what is required to setup this image up with Ember.js / Ember-CLI >= v3.0.

**Comment by danlynn** [comment]
Thanks!  I added a discussion of the CI env var and IE11 support to the [Important Change in ember-cli:3.0.0](https://github.com/danlynn/ember-cli#important-change-in-ember-cli300) section.  Let me know if this looks good.

**Comment by raido** [comment]
LGTM!
